charts/gateway-helm/templates/envoy-gateway-metrics-service.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" }}
```

Comment:
is there any strong reason for having this capabilities condition ?
for the case when this chart is installed before promethus is installed, this service wont get installed, which is not ideal, we ideally dont want to govern the order of install of components, so I'd vote to rm it

Reply:
+1